tests/cypress/integration/aliases_spec.js

```diff
@@ -0,0 +1,141 @@
+describe('Test aliases to H.O.C. (default)', () => {
```

Comment:
tnstead of "(default)" better to say something like "(no language cookie set)"

Also, if I read the describe and the test, I would see 
```
Test aliases to H.O.C. (default) when accessing H.O.C. home
```
which is not really telling me what we are testing. I suggest you change this describe, and the others in this file, to `Test aliases: loads English with no cookie set`
so then, each test will read something like
```
Test aliases: loads English with no cookie set when accessing H.O.C. home
```